mobile/src/main/java/org/openhab/habdroid/ui/WidgetAdapter.kt

```diff
@@ -562,21 +563,28 @@ class WidgetAdapter(
 
     class SwitchViewHolder internal constructor(initData: ViewHolderInitData) :
         LabeledItemBaseViewHolder(initData, R.layout.widgetlist_switchitem, R.layout.widgetlist_switchitem_compact) {
-        private val switch: CompoundButton = itemView.findViewById(R.id.toggle)
+        private val switch: MaterialSwitch = itemView.findViewById(R.id.toggle)
         private var isBinding = false
 
         init {
             switch.setOnCheckedChangeListener { _, checked ->
                 if (!isBinding) {
                     connection.httpClient.sendItemCommand(boundWidget?.item, if (checked) "ON" else "OFF")
                 }
+                switch.thumbIconDrawable = null
```

Comment:
This shouldn't be required as `bind()` should be called when the value of an Item changes, right? But for `SwitchViewHolder` `bind()` isn't called when pressing on the switch.

Reply:
The sequence should be
- switch is used
- -> command sent to server
- -> SSE event with new state sent by server
- SSE event causes updateWidget() -> updateWidgetAtPosition() -> notifyItemChanged()
- -> onBindViewHolder() is called, which in turn calls bind()